addons/web/static/src/core/utils/scrolling.js

```diff
@@ -192,3 +192,9 @@ export function getScrollingElement(document = window.document) {
     }
     return baseScrollingElement;
 }
+
+export function getScrollingTarget(contextItem = window.document) {
```

Comment:
window.document === document ?

Reply:
rename contextItem => scrollingElement

Reply:
`document` is not defined there, so we need to use `window.document` :slightly_smiling_face: (see the function just above that uses it too)